sdk/python/feast/ui/public/projects-list.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Is this file needed at all, if the `feast ui` command is generating it on the fly?

Reply:
I changed this because to generate it on the fly, I need to hard-code a host + port in that initial React component initialization, which wouldn't work properly